invenio_records_resources/service/search/query.py

```diff
@@ -26,13 +29,13 @@ def __init__(self, query_parser=None, **kwargs):
         self.query_parser = query_parser or _default_parser
         self.params = kwargs
 
-    def parse(self, queryparser):
+    def parse(self, querystring):
```

Comment:
Hmm the `querystring` term is a little overloaded: https://en.wikipedia.org/wiki/Query_string .. It means the whole part of the URL after the `?` e.g., for `https://example.com/?q=tag:good+tag:alright&lang=fr` it's `q=tag:good+tag:alright&lang=fr`, but in the code here querystring is referring to `tag:good+tag:alright` right? What if we used the term `searchstring` to resolve this ambiguity? (other names are possible too).